File: baseandroid/src/main/java/com/ocse/baseandroid/retrofit/ApiRetrofitManager.java
package com.ocse.baseandroid.retrofit;

import com.ocse.baseandroid.retrofit.base.BaseRetrofit;
import com.ocse.baseandroid.utils.ToastUtil;

public class ApiRetrofitManager {
    public static void init(String baseUrl) {
        if ("/".equals(baseUrl.substring(baseUrl.length()-1))){
            BaseRetrofit.getInstance().setURL(baseUrl);
        }else {
            ToastUtil.Companion.show("必须以/结尾");
        }

    }
}

File: baseandroid/src/main/java/com/ocse/baseandroid/retrofit/ApiRetrofitManager.kt
package com.ocse.baseandroid.retrofit

import com.ocse.baseandroid.retrofit.base.BaseRetrofit.Companion.instance
import com.ocse.baseandroid.utils.ToastUtil.Companion.show

/**
 * @author hujiayi
 */
object ApiRetrofitManager {
    fun init(baseUrl: String) {
        try {
            if ("/" == baseUrl.substring(baseUrl.length - 1)) {
                instance!!.uRL = baseUrl
            } else {
                show("必须以/结尾")
            }
        } catch (e: Exception) {
            show(e.message)
        }
    }
}